src/Components/README.md

```diff
@@ -163,3 +163,22 @@ builder.AddAzureServiceBus(settings =>
 ```
 
 - If both secret and passwordless mechanisms are configured, the secret credential overrides the passwordless identity setting.
+
+## Checklist for new Components
+
+New Components MUST have:
```

Comment:
Do we want to say something about what we don't want? Like the fact we try to *avoid* adding new APIs on top of the client APIs?

Reply:
Probably not in the checklist, but I can add something along those lines above.